values.yaml

```diff
@@ -15,7 +15,7 @@ docker:
 ##
 cog:
   name: cog
-  serviceType: ClusterIP
+  serviceType: ClusterIP # If using an Ingress, this must be NodePort
```

Comment:
This only needs to be `NodePort` if using the GCE default load balancer, rather than an alternative ingress controller like nginx.